dymos/transcriptions/common/test/test_polynomial_control_group.py

```diff
@@ -211,8 +218,6 @@ def test_polynomial_control_group_scalar_radau(self):
                               promotes_inputs=['*'],
                               promotes_outputs=['*'])
 
-        # p.model.connect('dt_dstau', 'control_interp_comp.dt_dstau')
```

Comment:
There's 3 other occurrences of this line in the file. Delete all of them?

Reply:
Removed.